README.md

```diff
@@ -1,13 +1,13 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_tree-from-object-DOM/)
+    - [DEMO LINK](https://ShapovalDenys.github.io/js_tree-from-object-DOM/)
```

Comment:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/38065883/172676094-3615ceef-693c-4ec6-87af-d99e21a2a4ed.png">
There should be no empty `ul` tags everywhere